sql/mysqld.cc

```diff
@@ -4267,6 +4273,9 @@ SHOW_VAR com_status_vars[] = {
      (char *)offsetof(System_status_var,
                       com_stat[(uint)SQLCOM_RELEASE_SAVEPOINT]),
      SHOW_LONG_STATUS, SHOW_SCOPE_ALL},
+    {"release_explict_snapshot",
```

Comment:
It'll be fine to fix this typo: release_explicit_snapshot.

Reply:
Done